src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -171,34 +181,26 @@ use it, first install Lighthouse from npm:
 **Example** - using `ChromeLauncher` to launch Headless
 
 ```javascript
-const {ChromeLauncher} = require('lighthouse/lighthouse-cli/chrome-launcher');
+const chromeLauncher = require('lighthouse/chrome-launcher/chrome-launcher');
 
 /**
- * Launches a debugging instance of Chrome on port 9222.
+ * Launches a debugging instance of Chrome.
  * @param {boolean=} headless True (default) to launch Chrome in headless mode.
- *     Set to false to launch Chrome normally.
+ *     False launches a full version of Chrome.
```

Comment:
My English teachers would hang their head in shame. Can you please give a specific example of how they should both look?